tsconfig.json

```diff
@@ -7,6 +7,6 @@
     "lib": ["ESNext", "DOM"],
     "noUncheckedIndexedAccess": true,
     "verbatimModuleSyntax": true, // Can be removed after updating nuxt to v3.8
-    "exactOptionalPropertyTypes": true
+    "exactOptionalPropertyTypes": false // disabled because of https://github.com/vuejs/core/issues/6532
```

Comment:
What's bugging me is why we haven't had this issue before ... Do you know what changes are causing this issue?

Reply:
no 🤷